### PR TITLE
Tackle 1 install fix

### PR DESCRIPTION
**Pull request by kfrankli**
Resolved issues with tackle 1 install
* Added instructions to match tackle 2 coverage of minikube
* Fixed issue with pulling wrong version of operator that would result in installation of 2.0 rather than 1.2
* Generally formatting improvements and consistency matching to tackle 2 instructions

### CODE REVIEW

File: content/Tackle/installation1.md [hunk 1]
```diff
@@ -3,14 +3,34 @@ title: "Installing Tackle 1.0"
...
+```
+
 
 ## Installing the Tackle Operator
 Follow the steps below to download and install the Tackle Operator on an Enterprise Kubernetes Platform cluster.
 
 **Prerequisites**
 
-* Cluster-admin privileges.
+* Cluster-admin privileges. This can be minikube
```
Comment (scoonRH) [inline content/Tackle/installation1.md]: Plz add a period.

File: content/Tackle/installation1.md [hunk 2]
```diff
@@ -37,12 +58,17 @@ Follow the steps below to install Tackle in a namespace by creating an instance
...
 1. Create an instance of the Tackle application, specifying its namespace:
 ```
-$ kubectl apply -n <namespace> -f https://raw.githubusercontent.com/konveyor/tackle-operator/main/src/main/resources/k8s/tackle/tackle.yaml
+$ kubectl apply -f https://raw.githubusercontent.com/konveyor/tackle-operator/main/src/main/resources/k8s/tackle/tackle.yaml -n <namespace>
 ```
 > **Note:** Multiple instances of the Tackle application can be created in the same namespace by specifying a unique name for each instance in the tackle.yaml file.
 
-2. Open the Kubernetes dashboard
-3. Click **Workloads** then **Deployments** to verify the installation.
+2. Verify the instance
```
Comment (scoonRH) [inline content/Tackle/installation1.md]: Plz add a period.

**Review by scoonRH — APPROVED** [review]

File: content/Tackle/installation1.md (hunk 1 above)
Comment (kfrankli) [reply content/Tackle/installation1.md]: resolved

File: content/Tackle/installation1.md (hunk 2 above)
Comment (kfrankli) [reply content/Tackle/installation1.md]: resolved

**Pull request merged**